WordPress/Classes/ViewRelated/Feature Highlight/Tooltip.swift

```diff
@@ -0,0 +1,290 @@
+import UIKit
+
+final class Tooltip: UIView {
+    private enum Constants {
+        static let leadingIconUnicode = "✨"
```

Comment:
Minor design nitpick, but there's a slight color difference with the designs. Here's a comparison:

Figma | Current Implementation
--|--
<img width="352" alt="Screen Shot 2022-05-12 at 18 57 26" src="https://user-images.githubusercontent.com/1299411/168069625-60021ca3-f21d-4b77-ac04-1fa0dd05b326.png"> | ![current](https://user-images.githubusercontent.com/1299411/168069486-401a13d6-5ee5-4bfa-a44d-e84f3a6b8190.png)

The design looks like it colors the emoji with `systemOrange`, but unfortunately in iOS we can't do that 😅  (at least to my knowledge). An alternative is to use SFSymbol's `sparkles`. We can apply custom colors over this, but it looks a bit different than the one used in the designs 😅 . Here's how it looks:

<img width="125" alt="Screen Shot 2022-05-12 at 18 58 55" src="https://user-images.githubusercontent.com/1299411/168069851-9506e151-4bef-4b59-a27d-f57ac9339e9d.png">
 
So I think there are two ways about this (and maybe we can consult Thomas for this):

1. Keep the ✨ emoji in its original color,
2. Use SFSymbol's `sparkles` and display it with `systemOrange`.

What do you think? @alpavanoglu , @iamthomasbishop

Reply:
@dvdchr @alpavanoglu I might've forgotten to mention this during design, apologies. We ran into this issue in the editor a couple of times, and I think we ended up using an image asset instead of SF for this so that both platforms could use the same image but I'd need to check Android to confirm.

Reply:
Just checked to confirm, and it looks like we're using the same image (presumably an SVG?) across platforms. For consistency, we should probably re-use that if possible.

Note: this is what it looks like in the Inserter menu on both platforms (the "sparkle" icon to indicate "new" blocks).

<a href="https://user-images.githubusercontent.com/1200432/168108251-e1e5b800-8046-4949-9c36-0d97866e4af8.png"><img src="https://user-images.githubusercontent.com/1200432/168108251-e1e5b800-8046-4949-9c36-0d97866e4af8.png" width="800"></a>

Reply:
Ah, TIL about the sparkle on the editor block! Thank you @iamthomasbishop 🙇🏼